### PR TITLE
No style image

### PR DESCRIPTION
**Pull request by Aaron-Fink**
fixed no image issue and eslint complaining

### CODE REVIEW

File: client/components/Overview/components/Sizes.jsx [hunk 1]
```diff
@@ -4,19 +4,31 @@ import React from 'react';
...
+    if (currentStyle.skus[key].quantity === 0) {
+      styleArray.push({ size: 'OUT OF STOCK' });
+    } else {
+      styleArray.push(currentStyle.skus[key]);
+    }
   }
 
   return (
     <SizesRender
       styleArray={styleArray}
```
Comment (greta-schock) [inline client/components/Overview/components/Sizes.jsx:24]: Opinion: would be easier to read if your props on lines 27-31 are ordered in the same way as lines 7-9. Doesn't affect anything, but it makes it easier to review

File: client/components/Overview/components/Styles.jsx [hunk 2]
```diff
@@ -3,13 +3,17 @@ import React, { useEffect, useState } from 'react';
...
+  id, currentStyle, setCurrentStyle, setCurrentSize, setCurrentQuantity,
+  setAddToCartNoSize, setValue,
+}) => {
   const [styles, setStyles] = useState([]);
 
   //  ajax request to get all styles for the current products id
   const getAllStyles = () => {
     axios.get('http://localhost:3000/api/getallstyles', { params: { id } })
       .then((results) => {
+        console.log(results);
```
Comment (greta-schock) [inline client/components/Overview/components/Styles.jsx:16]: Do you want to keep this console.log?

**Review by greta-schock — APPROVED** [review]
Minor comments added. Looks good!

File: client/components/Overview/components/Styles.jsx (hunk 2 above)
Comment (Aaron-Fink) [reply client/components/Overview/components/Styles.jsx:16]: yeah looking at it for image gallery

File: client/components/Overview/components/Sizes.jsx (hunk 1 above)
Comment (Aaron-Fink) [reply client/components/Overview/components/Sizes.jsx:24]: I will make a ticket to fix that.

**Pull request merged**